front/src/views/Transfers/ShipmentsOverview/ShipmentsOverviewView.tsx

```diff
@@ -30,7 +30,7 @@ function ShipmentsOverviewView() {
   // transform shipments data for UI
   const graphqlToTableTransformer = (shipmentQueryResult: ShipmentsQuery | undefined) =>
     shipmentQueryResult?.shipments
-      .filter((shipment) => shipment.sourceBase.id === baseId)
+      .filter((shipment) => shipment.sourceBase.id === baseId && shipment.targetBase.id === baseId)
```

Comment:
Sorry, I didn't clarify; you need || here

Reply:
That's actually my bad, thanks for the catch!